pkg/kapp/cmd/app/deploy.go

```diff
@@ -159,6 +159,11 @@ func (o *DeployOptions) Run() error {
 		return err
 	}
 
+	// if flag is set, kapp will just print the output and exit
+	if o.ApplyFlags.ApplyingChangesOpts.Run {
```

Comment:
it's a bit weird to use this boolean twice. first time things are printed, second time we just exist. how can we consolidate, so it's more linear?

Reply:
updated and now it is being used at one place only.